README.md

```diff
@@ -1,123 +1,69 @@
-
-We’re making our investigation and prototype public to provide another example for the upcoming discussion in the “GPU for the Web” W3C community group.
-
-NXT currently has the following features:
-
-- Command buffers, graphics and compute pipelines
-- Textures, samplers, vertex / index / uniform / storage buffers.
-- Descriptor sets (called bind groups) and push constants
-- SPIRV for the shading language
-- Validation
-
-NXT is missing a lot of things to be usable for anything else than prototyping:
-
-- Render-targets / render passes
-- Most of the fixed function pipeline state
-- Barriers / resource transitions and GPU - CPU synchronization
-- Buffer mapping
-- ...
-
-We chose to use SPIRV in our prototype because it was the only language that had translators to other shading languages, thanks to SPIRV-Cross which saved us a ton of work. [SPIRV-Cross](https://github.com/KhronosGroup/SPIRV-Cross) didn’t have an HLSL backend so we didn’t attempt a D3D12 backend (and D3D11 was too limiting for our prototype API). Only the Metal and the OpenGL backends are functional at this time. The OpenGL backend let us integrate in Chromium very easily.
-
-## Links
-
-An [overview document](https://docs.google.com/document/d/1-lAvR9GXaNJiqUIpm3N2XuGUWv_JrkpGizDN0bNq7wY) of the goals and design of NXT.
-
-Some of the investigations we made on the design of potential backend APIs:
-
-- [Binding model investigation](https://drive.google.com/open?id=1_xeTnk6DlN7YmePQQAlnHndA043rgwBzUYtatk5y7kQ)
-- [Data uploads investigation](https://drive.google.com/open?id=1Mi9l14zG8HzJ5Z6107SdPhON0mq4d-3SUI8iS631nek)
-- [Resource creation investigation](https://drive.google.com/open?id=1hK1SkTFkXJXPjyla0EEl1fOIwJSc6T41AV2mGiovyFU)
-- [Vertex setup investigation](https://drive.google.com/open?id=1SIUpdg-6Xm5FFF1ktdBfnR5oRKjyPAfXir7Drui4cYM)
-
-[Another presentation](https://drive.google.com/open?id=1mLQEM__twfivV7nJLDBIomS9pegOYkJQWyM6lTse4PQ) about our work with more details on the architecture of the prototype, and a [video](https://youtu.be/ThlZ5K4hJvo) of the demo we showed.
-
-TODO: add a link to the NXT-chromium repo once it is uploaded.
-
-## Key elements of the prototype’s architecture
-
-### Builder pattern for object creation
-
-In NXT, object creation is done through builder objects that gather initialization parameters with a fluent interface and return the initialized object when GetResult() is called. 
-
-In addition to the improved type-safety and subjective prettiness compared to giant constructors, this style enables additional optimizations. For example this removes the need for any check for an object being built and allows backend to forget parameters it might not care about.
+```sh
```

Comment:
?

Reply:
This is markdown to give the language to use for syntax highlighting. Here "shell" syntax.